services/text-formatters.js

```diff
@@ -119,6 +119,9 @@ function formatDate(d) {
 }
 
 function formatRelativeDate(timestamp) {
+  if (timestamp instanceof Date) {
+    timestamp = timestamp.getTime() / 1000
+  }
```

Comment:
I think it would actually be clearer/less "magic" if we did something like

```js
function formatRelativeDate(date) {
  return formatRelativeTimestamp(date.getTime() / 1000)
}

function formatRelativeTimestamp(timestamp) {
  return moment()
    .to(moment.unix(parseInt(timestamp, 10)))
    .toLowerCase()
}
```

and switch `date.service.js` (the only other thing using this helper) to use `formatRelativeTimestamp()`